apps/email/js/cards/compose.js

```diff
@@ -451,6 +467,37 @@ return [
       return bubble;
     },
 
+    getExistingEntriesForAutocomplete: function(inputNode) {
```

Comment:
This was just me being careful and not knowing the expected rules for email addressing. Changed to include all entries attached to the email, not just the current input node. Interface changed to not pass the inputNode.